Formula/no-more-secrets.rb

```diff
@@ -0,0 +1,16 @@
+class NoMoreSecrets < Formula
+  desc 'Recreates the SETEC ASTRONOMY effect from "Sneakers"'
```

Comment:
In my opinion, the inner quotes aren't even needed. After all it's *Sneakers* and not *sneakers*, so the quotes don't convey any additional information.